MeetupPlatformApi/Migrations/20220401140258_AddFluentApiForRefreshToken.cs

```diff
@@ -0,0 +1,247 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class AddFluentApiForRefreshToken : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Please, squash all migrations in the PR before it's merge.
You can do this by deleting and re-creating them. The purpose is to not have too many migrations (less we have, easier it would be to navigate through them).